applications/StructuralMechanicsApplication/tests/cl_test/AnisotropyCube/anisotropy_test_materials.json

```diff
@@ -0,0 +1,41 @@
+{
+    "properties" : [{
+        "model_part_name" : "Structure.Parts_Parts_Auto1",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "GenericAnisotropic3DLaw"
+            },
+            "Variables"        : {
+                "YOUNG_MODULUS_X" : 40e9,
+				"YOUNG_MODULUS_Y" : 10e9,
+				"YOUNG_MODULUS_Z" : 10e9,
+                "POISSON_RATIO_XY" : 0.2,
+				"POISSON_RATIO_XZ" : 0.2,
```

Comment:
didn't we agree on using a vector for this input?

Reply:
yes I forgot to update the test, sorry. Now we have the two capabilities: input as vector or one by one

Reply:
i would not allow two different inputs... let's agree on one and leave it as the only option.
I defend stronglt the vector input for efficiency, but if you have strong reasons in favor of the "scalar" input we can discuss.
In any case i would not leave two equivalent options since it would only make maintenance more complicated

Reply:
mm okkey, I do not really have any inconvenient let's allow only the vector version